ui/src/kapacitor/components/KapacitorRule.js

```diff
@@ -150,7 +163,7 @@ class KapacitorRule extends Component {
       <div className="page">
         <RuleHeader
           source={source}
-          onSave={isEditing ? this.handleEdit : this.handleCreate}
+          onSave={ruleID === 'new' ? this.handleCreate : this.handleEdit}
```

Comment:
i notice here and in multiple other cases that `'new'` is being used for comparison -- i haven't tested the code but just want to make sure that this is correct, or whether it should be `DEFAULT_RULE_ID`?

Reply:
ooh thanks for catching that.

Reply:
The cause of this was the same thing as the first comment you made below. so both are addressed.

Reply:
i'm not sure what you mean or which comment you're referring to, but i just wanted to bring your attention to this and the other `'new'` uses just in case.